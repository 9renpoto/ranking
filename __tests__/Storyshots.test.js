// https://github.com/storybooks/storybook/issues/1852
const initStoryshots = require("@storybook/addon-storyshots").default;
initStoryshots();
