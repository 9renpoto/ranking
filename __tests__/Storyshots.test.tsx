import initStoryshots, {
  multiSnapshotWithOptions
} from '@storybook/addon-storyshots'
import styleSheetSerializer from 'jest-styled-components/src/styleSheetSerializer'
import { addSerializer } from 'jest-specific-snapshot'

addSerializer(styleSheetSerializer)

initStoryshots({
  storyKindRegex: /^((?!.*?Dont Test).)*$/,
  storyNameRegex: /^((?!.*?Dont Test).)*$/,
  framework: 'react',
  suite: 'FileProperties',
  test: multiSnapshotWithOptions({})
})
