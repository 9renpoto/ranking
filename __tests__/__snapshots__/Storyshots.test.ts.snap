// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots atoms Avatar 1`] = `
<ForwardRef(styled.div)
  height={20}
  width={20}
/>
`;

exports[`Storyshots atoms Box 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.div)>
    port eyeballs Rufiyaa
  </ForwardRef(styled.div)>
  <ForwardRef(styled.div)>
    sensor users Administrator
  </ForwardRef(styled.div)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots atoms Button 1`] = `
<ForwardRef(styled.button)>
  Metical
</ForwardRef(styled.button)>
`;

exports[`Storyshots atoms Hero 1`] = `
<ForwardRef(styled.div)>
  Cotton harness
</ForwardRef(styled.div)>
`;

exports[`Storyshots atoms RankBar 1`] = `<DefaultRankBar />`;

exports[`Storyshots molecules Admire 1`] = `
<ForwardRef(styled.div)
  fixed={true}
>
  <ForwardRef(styled.div)
    fixed={true}
  >
    <DefaultAdmire />
    <DefaultAdmire />
    <DefaultAdmire />
    <DefaultAdmire />
  </ForwardRef(styled.div)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots molecules FixedButton 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.button)>
    Credit Card Account
  </ForwardRef(styled.button)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots molecules Hero 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.div)>
    <ForwardRef(styled.div)
      height={4.688}
      isCircle={true}
      width={4.688}
    >
      <img
        alt="help-desk"
        src="http://lorempixel.com/300/300"
      />
    </ForwardRef(styled.div)>
    <ForwardRef(styled.p)
      padding={1}
      white={true}
    >
      Balanced
    </ForwardRef(styled.p)>
  </ForwardRef(styled.div)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots molecules Range 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <p>
        Yen
      </p>
      <p>
        Credit Card Account
      </p>
    </ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <ForwardRef(styled.div)
        style={
          Object {
            "left": "50%",
          }
        }
      />
      <ForwardRef(styled.span)
        style={
          Object {
            "left": "50%",
          }
        }
      >
        3
      </ForwardRef(styled.span)>
    </ForwardRef(styled.div)>
  </ForwardRef(styled.div)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots molecules Range/input 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <p>
        deposit
      </p>
      <p>
        initiatives
      </p>
    </ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <ForwardRef(styled.input)
        step="25"
        type="range"
      />
      <ForwardRef(styled.span)
        style={
          Object {
            "left": "50%",
          }
        }
      >
        3
      </ForwardRef(styled.span)>
    </ForwardRef(styled.div)>
  </ForwardRef(styled.div)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots molecules Rank 1`] = `
<ForwardRef(styled.div)>
  <span>
    8372
  </span>
  <ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <ForwardRef(styled.div)
        height={2.5}
        width={2.5}
      >
        <img
          alt="Poland"
          src="http://lorempixel.com/300/300"
        />
      </ForwardRef(styled.div)>
      <ForwardRef(styled.p)
        padding={0}
      >
        Israel
      </ForwardRef(styled.p)>
    </ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <p>
            withdrawal
          </p>
          <p>
            indexing
          </p>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.input)
            step="25"
            type="range"
          />
          <ForwardRef(styled.span)
            style={
              Object {
                "left": "50%",
              }
            }
          >
            3
          </ForwardRef(styled.span)>
        </ForwardRef(styled.div)>
      </ForwardRef(styled.div)>
    </ForwardRef(styled.div)>
  </ForwardRef(styled.div)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots molecules Result 1`] = `
<React.Fragment>
  <ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <span>
        50人中
      </span>
      10位
    </ForwardRef(styled.div)>
  </ForwardRef(styled.div)>
  <ForwardRef(styled.p)>
    monitor
  </ForwardRef(styled.p)>
</React.Fragment>
`;

exports[`Storyshots pages Admire 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.nav)>
    <ul>
      <li>
        <a
          className="active"
          href="#"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  448,
                  512,
                  Array [],
                  "f007",
                  "M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z",
                ],
                "iconName": "user",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  576,
                  512,
                  Array [],
                  "f044",
                  "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                ],
                "iconName": "edit",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
    </ul>
  </ForwardRef(styled.nav)>
  <ForwardRef(styled.main)>
    <ForwardRef(styled.div)
      fixed={true}
    >
      <ForwardRef(styled.div)
        fixed={true}
      >
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)
              height={2.5}
              width={2.5}
            >
              <img
                alt="Money Market Account"
                src="http://lorempixel.com/300/300"
              />
            </ForwardRef(styled.div)>
            <ForwardRef(styled.p)
              padding={0}
            >
              system
            </ForwardRef(styled.p)>
          </ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <p>
                  monitoring
                </p>
                <p>
                  dynamic
                </p>
              </ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <ForwardRef(styled.input)
                  step="25"
                  type="range"
                />
                <ForwardRef(styled.span)
                  style={
                    Object {
                      "left": "50%",
                    }
                  }
                >
                  3
                </ForwardRef(styled.span)>
              </ForwardRef(styled.div)>
            </ForwardRef(styled.div)>
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)
              height={2.5}
              width={2.5}
            >
              <img
                alt="Enterprise-wide"
                src="http://lorempixel.com/300/300"
              />
            </ForwardRef(styled.div)>
            <ForwardRef(styled.p)>
              zero defect
            </ForwardRef(styled.p)>
          </ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <p>
                  invoice
                </p>
                <p>
                  Licensed
                </p>
              </ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <ForwardRef(styled.input)
                  step="25"
                  type="range"
                />
                <ForwardRef(styled.span)
                  style={
                    Object {
                      "left": "50%",
                    }
                  }
                >
                  3
                </ForwardRef(styled.span)>
              </ForwardRef(styled.div)>
            </ForwardRef(styled.div)>
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)
              height={2.5}
              width={2.5}
            >
              <img
                alt="web-readiness"
                src="http://lorempixel.com/300/300"
              />
            </ForwardRef(styled.div)>
            <ForwardRef(styled.p)>
              initiatives
            </ForwardRef(styled.p)>
          </ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <p>
                  Avon
                </p>
                <p>
                  markets
                </p>
              </ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <ForwardRef(styled.input)
                  step="25"
                  type="range"
                />
                <ForwardRef(styled.span)
                  style={
                    Object {
                      "left": "50%",
                    }
                  }
                >
                  3
                </ForwardRef(styled.span)>
              </ForwardRef(styled.div)>
            </ForwardRef(styled.div)>
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)
              height={2.5}
              width={2.5}
            >
              <img
                alt="success"
                src="http://lorempixel.com/300/300"
              />
            </ForwardRef(styled.div)>
            <ForwardRef(styled.p)>
              deliverables
            </ForwardRef(styled.p)>
          </ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <p>
                  International
                </p>
                <p>
                  Optimization
                </p>
              </ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <ForwardRef(styled.input)
                  step="25"
                  type="range"
                />
                <ForwardRef(styled.span)
                  style={
                    Object {
                      "left": "50%",
                    }
                  }
                >
                  3
                </ForwardRef(styled.span)>
              </ForwardRef(styled.div)>
            </ForwardRef(styled.div)>
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)
              height={2.5}
              width={2.5}
            >
              <img
                alt="Berkshire"
                src="http://lorempixel.com/300/300"
              />
            </ForwardRef(styled.div)>
            <ForwardRef(styled.p)>
              Home Loan Account
            </ForwardRef(styled.p)>
          </ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <p>
                  Plastic
                </p>
                <p>
                  initiative
                </p>
              </ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <ForwardRef(styled.input)
                  step="25"
                  type="range"
                />
                <ForwardRef(styled.span)
                  style={
                    Object {
                      "left": "50%",
                    }
                  }
                >
                  3
                </ForwardRef(styled.span)>
              </ForwardRef(styled.div)>
            </ForwardRef(styled.div)>
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)
              height={2.5}
              width={2.5}
            >
              <img
                alt="infomediaries"
                src="http://lorempixel.com/300/300"
              />
            </ForwardRef(styled.div)>
            <ForwardRef(styled.p)>
              JSON
            </ForwardRef(styled.p)>
          </ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <p>
                  Configurable
                </p>
                <p>
                  Berkshire
                </p>
              </ForwardRef(styled.div)>
              <ForwardRef(styled.div)>
                <ForwardRef(styled.input)
                  step="25"
                  type="range"
                />
                <ForwardRef(styled.span)
                  style={
                    Object {
                      "left": "50%",
                    }
                  }
                >
                  3
                </ForwardRef(styled.span)>
              </ForwardRef(styled.div)>
            </ForwardRef(styled.div)>
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
      </ForwardRef(styled.div)>
      <ForwardRef(styled.div)>
        <ForwardRef(styled.button)>
          Manager
        </ForwardRef(styled.button)>
      </ForwardRef(styled.div)>
    </ForwardRef(styled.div)>
  </ForwardRef(styled.main)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots pages Mypage 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.nav)>
    <ul>
      <li>
        <a
          className="active"
          href="#"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  448,
                  512,
                  Array [],
                  "f007",
                  "M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z",
                ],
                "iconName": "user",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  576,
                  512,
                  Array [],
                  "f044",
                  "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                ],
                "iconName": "edit",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
    </ul>
  </ForwardRef(styled.nav)>
  <ForwardRef(styled.main)>
    <ForwardRef(styled.div)>
      <ForwardRef(styled.div)>
        <ForwardRef(styled.div)
          height={4.688}
          isCircle={true}
          width={4.688}
        >
          <img
            alt="Jamaica"
            src="http://lorempixel.com/300/300"
          />
        </ForwardRef(styled.div)>
        <ForwardRef(styled.p)
          padding={1}
          white={true}
        >
          Auto Loan Account
        </ForwardRef(styled.p)>
      </ForwardRef(styled.div)>
    </ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <p>
                とても悪い
              </p>
              <p>
                とても良い
              </p>
            </ForwardRef(styled.div)>
            <ForwardRef(styled.div)>
              <ForwardRef(styled.div)
                style={
                  Object {
                    "left": "50%",
                  }
                }
              />
              <ForwardRef(styled.span)
                style={
                  Object {
                    "left": "50%",
                  }
                }
              >
                3
              </ForwardRef(styled.span)>
            </ForwardRef(styled.div)>
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
      </ForwardRef(styled.div)>
      <ForwardRef(styled.div)>
        <ForwardRef(styled.div)>
          <ForwardRef(styled.div)>
            <span>
              50人中
            </span>
            10位
          </ForwardRef(styled.div)>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.p)>
          Granite
        </ForwardRef(styled.p)>
      </ForwardRef(styled.div)>
    </ForwardRef(styled.div)>
    <ForwardRef(styled.div)>
      <ForwardRef(styled.div)>
        <ForwardRef(styled.div)
          style={
            Object {
              "width": "83%",
            }
          }
        >
          <ForwardRef(styled.div)
            height={0.938}
            width={0.938}
          >
            <img
              alt="multi-byte"
              src="http://lorempixel.com/300/300"
            />
          </ForwardRef(styled.div)>
          <ForwardRef(styled.p)
            padding={0}
            white={true}
          >
            intermediate
          </ForwardRef(styled.p)>
        </ForwardRef(styled.div)>
        <ForwardRef(styled.div)
          style={
            Object {
              "width": "43%",
            }
          }
        >
          <ForwardRef(styled.div)
            height={0.938}
            width={0.938}
          >
            <img
              alt="clicks-and-mortar"
              src="http://lorempixel.com/300/300"
            />
          </ForwardRef(styled.div)>
          <ForwardRef(styled.p)
            padding={0}
            white={true}
          >
            focus group
          </ForwardRef(styled.p)>
        </ForwardRef(styled.div)>
      </ForwardRef(styled.div)>
    </ForwardRef(styled.div)>
  </ForwardRef(styled.main)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots pages Signin 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.main)>
    <form
      action="#"
      method="post"
    >
      <ForwardRef(styled.div)>
        <ForwardRef(styled.input)
          placeholder="Marty_Parisian26@yahoo.com"
          type="email"
        />
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Object {
              "icon": Array [
                512,
                512,
                Array [],
                "f0e0",
                "M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z",
              ],
              "iconName": "envelope",
              "prefix": "fas",
            }
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </ForwardRef(styled.div)>
      <ForwardRef(styled.div)>
        <ForwardRef(styled.input)
          placeholder="Berkshire"
          type="password"
        />
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon={
            Object {
              "icon": Array [
                448,
                512,
                Array [],
                "f09c",
                "M400 256H152V152.9c0-39.6 31.7-72.5 71.3-72.9 40-.4 72.7 32.1 72.7 72v16c0 13.3 10.7 24 24 24h32c13.3 0 24-10.7 24-24v-16C376 68 307.5-.3 223.5 0 139.5.3 72 69.5 72 153.5V256H48c-26.5 0-48 21.5-48 48v160c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V304c0-26.5-21.5-48-48-48z",
              ],
              "iconName": "unlock",
              "prefix": "fas",
            }
          }
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size={null}
          spin={false}
          symbol={false}
          title=""
          transform={null}
        />
      </ForwardRef(styled.div)>
      <ForwardRef(styled.button)
        disabled={true}
      >
        Sign in
      </ForwardRef(styled.button)>
    </form>
  </ForwardRef(styled.main)>
</ForwardRef(styled.div)>
`;

exports[`Storyshots templates Default 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.nav)>
    <ul>
      <li>
        <a
          className="active"
          href="#"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  448,
                  512,
                  Array [],
                  "f007",
                  "M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z",
                ],
                "iconName": "user",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  576,
                  512,
                  Array [],
                  "f044",
                  "M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z",
                ],
                "iconName": "edit",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size={null}
            spin={false}
            symbol={false}
            title=""
            transform={null}
          />
        </a>
      </li>
    </ul>
  </ForwardRef(styled.nav)>
  <ForwardRef(styled.main) />
</ForwardRef(styled.div)>
`;

exports[`Storyshots templates SignIn 1`] = `
<ForwardRef(styled.div)>
  <ForwardRef(styled.main) />
</ForwardRef(styled.div)>
`;
